### DISCUSSION TITLE
PeeringDB API / IX-API

### DISCUSSION THREAD

**Discussion by daverdzx** [peering-manager/peering-manager#620] [Q&A] [answered]
Hello Guys, I was trying to connect to PeeringDB API via IX-API on PM but I am getting a "404 not found" error as shown in the image.  Is there a way to avoid the authentication via token or to skip the '/auth/token' part in the URL as this '/auth/token' part is being attached by the PM UI and as far as I have tested it seems like it does not exist as part of the PeeringDB API methods hence the 404 error.  Or maybe I am using an incorrect URL (https://www.peeringdb.com/api), if this is the case can you provide the correct URL or approach to connect with PDB.  

![image](https://user-images.githubusercontent.com/111540778/186288736-b0cc7b09-f944-477c-91c9-9b4773d6c315.png)

By the way:  I've filled the "API-key" and "Identity" fields with my PDB token and user respectively (for the API-secret field I dont know what to do with this), and if I try to left this fields empty it says is a required field...

![image](https://user-images.githubusercontent.com/111540778/186290927-5a28f0e9-5260-4a0f-953f-c69f50b81ce7.png)

Please help, thank you so much.

**Comment by Paktosan** [comment, marked as answer]
tl;dr: See https://peering-manager.readthedocs.io/en/latest/configuration/optional-settings/#peeringdb_api_key

I think, you misunderstood, what IX-API is.
IX-API is something an exchange offers, while PeeringDB is a user managed database not attached to any single exchange.
For configuring PeeringDB, please check out the link above for the required settings.

**Reply by daverdzx** [reply]
Hello Guys/Julian, thanks for your support.  Sorry for the confusion there I forgot to post, but yes a couple of days ago after trial and error and reading the documentation I realized that PDB and IX-API are two different things, and finally made it work.
Have a great day.
